public/app/features/alerting/unified/components/rules/RuleActionsButtons.tsx

```diff
@@ -123,6 +127,21 @@ export const RuleActionsButtons = ({ rule, rulesSource }: Props) => {
           />
         </Tooltip>
       );
+
+      moreActions.push(
+        <MenuItemPauseRule
+          rule={rule}
+          onPauseChange={async () => {
+            // Uses INSTANCES_DISPLAY_LIMIT + 1 here as exporting LIMIT_ALERTS from RuleList has the side effect
+            // of breaking some unrelated tests in Policy.test.tsx due to mocking approach
+            const limitAlerts = hasActiveFilters ? undefined : INSTANCES_DISPLAY_LIMIT + 1;
+            // Trigger a re-fetch of the rules table
+            // TODO: Migrate rules table functionality to RTK Query, so we instead rely
+            // on tag invalidation (or optimistic cache updates) for this
+            await dispatch(fetchAllPromAndRulerRulesAction(false, { limitAlerts }));
```

Comment:
nit: Probably not necessary to await the dispatch here

Reply:
Oops - I semi-blindly copied from [here](https://github.com/grafana/grafana/blob/649c456eab6024bb13eb0c38372454839fb7e9ff/public/app/features/alerting/unified/RuleList.tsx#L85), but you're right that we don't need to await that. Have removed it from here (left the other instance untouched though)